react-components/src/components/Minicart/__test__/couponForm.test.js

```diff
@@ -39,37 +44,13 @@ describe('<CouponForm />', () => {
 
         const { asFragment } = render(
             <I18nextProvider i18n={i18n}>
-                <CartProvider initialState={initialState} reducerFactory={() => state => state}>
-                    <CouponForm />
-                </CartProvider>
+                <MockedProvider mocks={[]}>
+                    <CartProvider initialState={initialState} reducerFactory={() => state => state}>
+                        <CouponForm />
+                    </CartProvider>
+                </MockedProvider>
             </I18nextProvider>
         );
         expect(asFragment()).toMatchSnapshot();
     });
-
-    it('applies an coupon', () => {
```

Comment:
Since we don't have E2E tests, I would like to keep the tests that make sure that clicks on "Apply coupon" calls the `useCouponForm` resp. "Remove coupon" calls the `useCouponItem` hooks.

Reply:
I was thinking of implementing some tests for the `useXXX` "hooks" I added, but you're right, we still need to test the wiring here.